strongbox-data-service/src/main/java/org/carlspring/strongbox/data/domain/GenericEntity.java

```diff
@@ -20,7 +20,7 @@
 {
 
     @Id
-    protected String id;
+    protected String databaseId;
```

Comment:
@AlexOreshkevich,

I don't think you've reverted the changes we discussed, regarding this. Or maybe you haven't pushed...?